src/query/storages/parquet/src/parquet_rs/parquet_reader/row_group.rs

```diff
@@ -317,3 +348,59 @@ impl<'a> RowGroups for InMemoryRowGroup<'a> {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
Databend unit test style is in a seprate dir `tests/it` of current crate, references:
[1] https://github.com/datafuselabs/databend/tree/23c29ce2cb4fae44c11ee3258ad0f230eb7cacc1/src/query/storages/hive/hive/tests/it
[2] https://github.com/datafuselabs/databend/tree/23c29ce2cb4fae44c11ee3258ad0f230eb7cacc1/src/common/base/tests/it

[2]

Reply:
There is a unit test to reference:
https://github.com/datafuselabs/databend/blob/main/src/common/base/tests/it/range_merger.rs

Reply:
thanks, i would like to fix later.